app/controllers/users_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+    def show
+        @user = User.find_by id: params[:id]
+        unless @user
+        flash[:error] = "User not found."
+        redirect_to root_path and return
```

Comment:
mặc định line cuối cùng trong method là ruby tự return rồi em, ko cần khai báo
```suggestion
        redirect_to root_path
```